frontend/device/android/device.lua

```diff
@@ -352,4 +353,15 @@ end
 android.LOGI(string.format("Android %s - %s (API %d) - flavor: %s",
     android.prop.version, getCodename(), Device.firmware_rev, android.prop.flavor))
 
+--- Returns true if the file is a script we allow running
```

Comment:
Not needed. The function `Device:canExecuteScript()` does the same on all devices. We override its behaviour on android but there's no need to explain what the function does again.

See for example `Device:exit()`. It is explained in https://github.com/koreader/koreader/blob/master/frontend/device/generic/device.lua#L337-L343 but not again in https://github.com/koreader/koreader/blob/master/frontend/device/android/device.lua#L347-L350 (because that is just a device specific implementation of the exit function).

Also, I didn't notice on last review. Could you move the function before the log in 353?

Reply:
OK